backend/typescript/rest/reviewRoutes.ts

```diff
@@ -9,15 +9,19 @@ import {
 import { getErrorMessage, sendErrorResponse } from "../utilities/errorResponse";
```

Comment:
One of the ticket requirements is that an admin should not be able to modify the `createdBy` field for another review in a patch. Do we do that in this pr or will that be done in a different one?

Reply:
Looks like this feature is blocked by https://www.notion.so/uwblueprintexecs/16fb020b93f641c8b384f76a1b3c769e?v=aedc8d99cd8b437cb556ece10f3c0e3c&p=5c6374c816774d9688ea7b4d7aa6e4c0. We could either wait till that gets merged, rebase and then implement this or spin it off into a separate ticket (probably preferable?)

Reply:
probably better to spin it off into a separate ticket so this PR doesn't go stale